widgetry_demo/src/lib.rs

```diff
@@ -297,12 +297,167 @@ fn setup_scrollable_canvas(ctx: &mut EventCtx) -> Drawable {
 }
 
 fn make_controls(ctx: &mut EventCtx) -> Panel {
+    use widgetry::{ButtonBuilder, ButtonState};
     Panel::new(Widget::col(vec![
         Text::from_multiline(vec![
-            Line("widgetry demo").small_heading(),
+            Line("widgetry demo").big_heading_styled(),
             Line("Click and drag to pan, use touchpad or scroll wheel to zoom"),
         ])
         .draw(ctx),
+        // Button Style Gallery
+        Text::from(Line("Buttons").big_heading_styled().size(18)).draw(ctx),
+        Widget::row(vec![
+            Widget::col(vec![
+                Text::from(Line("Neutral Dark")).bg(Color::CLEAR).draw(ctx),
+                ButtonBuilder::new()
+                    .label_text("Primary")
+                    .label_color(Color::hex("#666666"), ButtonState::Default) // TODO is this the right color? it differs from icon and icon+text tinting
```

Comment:
It's kind of awkward that widgetry doesn't know about ColorScheme. 

*most* of our button building happens within map_gui or game, so it's fine. But for the widgetry demo and for the other *widget* which have inner buttons, I don't yet have a good solution.

Reply:
To be clear, I think it makes sense that our color schemes are defined outside of widgetry. There's just not yet a good way of applying an external color scheme to our other widgets (yet?).

Reply:
Either we could move more of the work to happen inside `widgetry/src/syle.rs`, making it sort of the color scheme but for UI things, or there might be some other way to plumb in theming for widgets. I'd say until we better know what other non map_gui users look like, fine like this.